crates/sui-node/src/lib.rs

```diff
@@ -805,6 +802,10 @@ impl SuiNode {
             cur_epoch_store.record_epoch_reconfig_start_time_metric();
             let _ = self.end_of_epoch_channel.send(next_epoch_committee.clone());
 
+            let new_epoch_store = self
+                .reconfigure_state(&cur_epoch_store, next_epoch_committee, system_state)
+                .await;
+
```

Comment:
@akichidis  is it safe to call this function before `narwhal_manager.shutdown().await;` (line 826)?

Reply:
I would say yes, but I would suggest not moving this now unless it is really necessary for some reason. I would feel much more safer knowing that we first shutdown Narwhal before we attempt any further reconfiguration.

Reply:
cool will revert this change